src/frontend/components/UI/Sidebar/components/HeroicVersion/index.tsx

```diff
@@ -54,55 +61,63 @@ export default React.memo(function HeroicVersion() {
   const version = heroicVersion
 
   return (
-    <>
-      {((showChangelogModal &&
-        !hideChangelogsOnStartup &&
-        heroicVersion !== lastChangelogShown) ||
-        showChangelogModalOnClick) && (
-        <ChangelogModal
-          dimissVersionCheck
-          onClose={() => {
-            setShowChangelogModal(false)
-            setShowChangelogModalOnClick(false)
-            setLastChangelogShown(heroicVersion)
-          }}
-        />
-      )}
-      <span
-        className="heroicVersion"
-        role="link"
-        title={t(
-          'info.heroic.click-to-see-changelog',
-          'Click to see changelog'
+    <div className="heroicVersionContainer" data-tour="sidebar-version">
+      <>
+        {((showChangelogModal &&
+          !hideChangelogsOnStartup &&
+          heroicVersion !== lastChangelogShown) ||
+          showChangelogModalOnClick) && (
+          <ChangelogModal
+            dimissVersionCheck
+            onClose={() => {
+              setShowChangelogModal(false)
+              setShowChangelogModalOnClick(false)
+              setLastChangelogShown(heroicVersion)
+            }}
+          />
         )}
-        onClick={() => setShowChangelogModalOnClick((current) => !current)}
-      >
-        <span className="heroicVersion__title">
-          <span>{t('info.heroic.version', 'Heroic Version')}: </span>
-        </span>
-        <strong>{version}</strong>
-      </span>
-      {shouldShowUpdates && (
-        <div className="heroicNewReleases">
-          <span>{t('info.heroic.newReleases', 'Update Available!')}</span>
-          {newStable && (
-            <a
-              title={newStable.tag_name}
-              onClick={() => window.api.openExternalUrl(newStable.html_url)}
-            >
-              {t('info.heroic.stable', 'Stable')} ({newStable.tag_name})
-            </a>
-          )}
-          {newBeta && (
-            <a
-              title={newBeta.tag_name}
-              onClick={() => window.api.openExternalUrl(newBeta.html_url)}
-            >
-              {t('info.heroic.beta', 'Beta')} ({newBeta.tag_name})
-            </a>
-          )}
+        <div className="heroicVersionWrapper">
+          <span
+            className="heroicVersion"
+            role="link"
+            title={t(
+              'info.heroic.click-to-see-changelog',
+              'Click to see changelog'
+            )}
+            onClick={() => setShowChangelogModalOnClick((current) => !current)}
+          >
+            <span className="heroicVersion__title">
+              <span>{t('info.heroic.version', 'Heroic Version')}: </span>
+            </span>
+            <strong>{version}</strong>
+          </span>
+          <TourButton
```

Comment:
this `(?)` button is not visible for collapsed sidebars, and it's making things wider than the actual sidebar width so icons are misaligned

<img width="92" alt="Screenshot 2025-03-09 at 7 00 15 PM" src="https://github.com/user-attachments/assets/51e915b4-6520-4d13-9630-53ef52d27880" />

Reply:
Should be fixed now

![image](https://github.com/user-attachments/assets/58092379-d8eb-4c4b-9733-210d4b81d38b)
![image](https://github.com/user-attachments/assets/eea8fce8-8e8f-4e8e-a633-1b554313287c)